fs/write_json.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Should we ignore any warning?

I didn't find the answer in the style guide.

But it seems that the other modules have not ignored it.